ingestion/functions/parsing/peru/peru_test.py

```diff
@@ -0,0 +1,47 @@
+
+import json
+import os
+import pytest
+import tempfile
+import pandas as pd
+
+from datetime import date
+
+_SOURCE_ID = "abc123"
+_SOURCE_URL = "https://foo.bar"
+_PARSED_CASE = (
+{'caseReference': {'sourceId': 'abc123',
```

Comment:
can you format that a bit nicer please? vscode if you're using it has a formatter (autopep8 that you can install separately) which should help.

Reply:
thanks for the tip!